cmd/infracost/run.go

```diff
@@ -74,14 +75,24 @@ func runMain(cmd *cobra.Command, runCtx *config.RunContext) error {
 		}
 	}
 
+	// Create a mutex for each path, so we can synchronize the runs of any
+	// projects that have the same path. This is necessary because Terraform
+	// can't run multiple operations in parallel on the same path.
+	pathMuxs := map[string]*sync.RWMutex{}
```

Comment:
Not sure if we need a `RWMutex` here - not a big issue tho. 

```suggestion
	pathMuxs := map[string]*sync.Mutex{}
```